submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/resources/SubmissionMetadataResource.kt

```diff
@@ -0,0 +1,59 @@
+package ac.uk.ebi.biostd.submission.web.resources
+
+import ac.uk.ebi.biostd.integration.SubFormat.Companion.TSV
+import ac.uk.ebi.biostd.submission.converters.BioUser
+import ac.uk.ebi.biostd.submission.domain.service.TempFileGenerator
+import ac.uk.ebi.biostd.submission.web.handlers.MetadataUpdateWebHandler
+import ac.uk.ebi.biostd.submission.web.model.FileSubmitWebRequest
+import ac.uk.ebi.biostd.submission.web.model.OnBehalfRequest
+import ebi.ac.uk.extended.model.FileMode.MOVE
+import ebi.ac.uk.model.Submission
+import ebi.ac.uk.model.constants.MULTIPART_FORM_DATA
+import ebi.ac.uk.model.constants.SUBMISSION
+import ebi.ac.uk.security.integration.model.api.SecurityUser
+import org.springframework.http.HttpHeaders
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.ResponseBody
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.multipart.MultipartFile
+
+@RestController
+@RequestMapping("/submissions")
+@PreAuthorize("isAuthenticated()")
+@Suppress("LongParameterList")
+class SubmissionMetadataResource(
+    private val tempFileGenerator: TempFileGenerator,
+    private val metadataUpdateWebHandler: MetadataUpdateWebHandler
+) {
+    // TODO Add support for the rest of submission extensions and direct page tab content
+    // TODO Add documentation for the update metadata endpoints
+    // TODO Add support for the update metadata operation in the CLI
+    @PutMapping(
+        value = ["/metadata/update"],
+        headers = ["${HttpHeaders.CONTENT_TYPE}=$MULTIPART_FORM_DATA"],
+        produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    @ResponseBody
+    fun updateMetadata(
```

Comment:
I was thinking we were going to implement detect metada change ourself instead of making ST or use choose wich approach to do. This looks not good for me, it looks like if email client has to options, write email and write email with attachtment.